core/src/main/java/tech/pegasys/web3signer/core/Eth1Runner.java

```diff
@@ -234,10 +238,28 @@ private MappedResults<ArtifactSigner> bulkLoadSigners(
           azureResult.getValues().size(),
           azureResult.getErrorCount());
       registerSignerLoadingHealthCheck(KEYS_CHECK_AZURE_BULK_LOADING, azureResult);
-      return azureResult;
-    } else {
-      return MappedResults.newSetInstance();
+      results = MappedResults.merge(results, azureResult);
     }
+
+    // wallet bulk loading
```

Comment:
nit: Think it deserves an extracted method instead of this comment

Reply:
done